release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1936,6 +1936,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+* If you are running a cluster on VMware using the `x86_64` architecture configured with the `platform: none` field in the `install-config.yaml` file, an upgrade from {product-title} cluster version 4.6 to version 4.7 might fail. This upgrade failure happens when a cluster uses virtual machines (VMs) configured with virtual hardware version 14 or greater. As a workaround, you can configure VMs with virtual hardware version 13. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1941714[*BZ#1941714*])
```

Comment:
This seems to be specific to VMware VMs.

Reply:
@jboxman right. The gotcha is some of our customers run `platform: none` on vSphere. They will still experience this issue with vmxnet3 v4 udp offload and dropped packets

Reply:
That's right. It affects clusters running on VMware that do not have a platform set in the install-config.